lib/matplotlib/cbook.py

```diff
@@ -783,8 +783,12 @@ def is_sequence_of_strings(obj):
     """
     if not iterable(obj):
         return False
-    if is_string_like(obj):
-        return False
+    if is_string_like(obj) and not isinstance(obj, np.ndarray):
```

Comment:
What does `is_string_like(series of str)` do? Does it loop through the array checking to see if each element is a `str`?